starwars/src/components/CharacterCard.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import Character from './Character'
+
+// components accept arbitary inputes called props and return React
+//elemnt descrbing what should appear on the function 
+const CharacterCard = props => {
+    return(
+        <div>
+            {props.data.map(char =>   //Rendering an Array of Data with map
+                <Character characterData={char} />
```

Comment:
You need to use a `key` attribute here.  `Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity`